### PR TITLE
refactor(Chat): move send message logic out of status chat input

### PR DESCRIPTION
**Pull request by PascalPrecht**
`StatusChatInput` ideally shouldn't rely on chatsModel and other global
objects at all. Also, when using the component in different places, it can cause
accidental sending of message when testing the component (because all the logic is already
wired up)

### CODE REVIEW

File: ui/app/AppLayouts/Chat/ChatColumn.qml [hunk 1]
```diff
@@ -268,6 +268,21 @@ StackLayout {
                 onStickerSelected: {
                     chatsModel.sendSticker(hashId, packId)
                 }
+                onSendMessage: {
+                    if (chatInput.fileUrls.length > 0){
+                        chatsModel.sendImage(chatInput.fileUrls[0]);
+                    }
+                    var msg = chatsModel.plainText(Emoji.deparse(chatInput.textInput.text).trim()).trim()
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/ChatColumn.qml:275]: FYI, this line was changed by Hydrogen. He removed the `trim`s since they caused issues in code blocks

**Review by jrainville — APPROVED** [review]
Looks good. I just left a comment on a potential conflict you might have

File: ui/app/AppLayouts/Chat/ChatColumn.qml (hunk 1 above)
Comment (PascalPrecht) [reply ui/app/AppLayouts/Chat/ChatColumn.qml:275]: uhm.. I believe the change isn't there yet, otherwise there'd be a conflict

Comment (jrainville) [reply ui/app/AppLayouts/Chat/ChatColumn.qml:275]: Indeed, you can see it here: https://github.com/status-im/nim-status-client/pull/1475

**Pull request merged**